spec/rspec/core/formatters/base_text_formatter_spec.rb

```diff
@@ -268,6 +268,7 @@ def run_all_and_dump_failures
   describe "custom_colors" do
     it "uses the custom success color" do
       RSpec.configure do |config|
+        config.no_color = false
         config.color = true
         config.tty = true
```

Comment:
Can we just set `config.force_color = true` here?  In general, it would be nice to remove/replace any uses of the `tty` option since we are doing away with it and `force_color` is the new way to force color.